### PR TITLE
rubocop compliance for modulesync_config 5.3.0

### PR DESCRIPTION
**Pull request by traylenator**
#### Pull Request (PR) description
rubocop corrections for 5.3.0

Includes the ignoring of RSpec/MultipleMemoizedHelpers which is very prevalent in this module.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -2,5 +2,6 @@
 # Managed by modulesync - DO NOT EDIT
 # https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
 
+inherit_from: .rubocop_todo.yml
```
Comment (smortex) [inline .rubocop.yml:5]: This will vanish on next modulesync update.

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+# Is very common in this module
+RSpec/MultipleMemoizedHelpers:
+  Enabled: false
```
Comment (smortex) [inline .rubocop_todo.yml:4]: I had to disable this from time to time, it does not work well with large code base. Maybe we can opt-out by default?

**Review by smortex — COMMENTED** [review]
LGTM

File: .rubocop.yml (hunk 1 above)
Comment (traylenator) [reply .rubocop.yml:5]: Looking at https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.rubocop.yml.erb#L5

```erb
<% if File.exist?(File.join(@metadata[:workdir], '.rubocop_todo.yml')) -%>
inherit_from: .rubocop_todo.yml
<% end -%>
```

I think not:

**Review by bastelfreak — APPROVED** [review]

File: .rubocop.yml (hunk 1 above)
Comment (smortex) [reply .rubocop.yml:5]: w00t! :heart_eyes:

**Review by smortex — APPROVED** [review]

**Pull request merged**